source/install/desktop.rst

```diff
@@ -71,3 +69,23 @@ To download the unofficial, community-driven .deb packages for the desktop appli
 2. Open a terminal and execute the command ``sudo dpkg -i mattermost-desktop-<VERSION>-<ARCH>.deb``
 
 3. Open **Dash** (located at top left corner) and input ``mattermost``, then click the Mattermost icon. The Mattermost desktop should open automatically.
+
+Troubleshooting
+--------------------------------------------------
+
+Possible solutions to issues encountered when using the Desktop App.
+
+"Installation has failed" dialog
```

Comment:
It seems like the headings are giving a lot of visual priority to errors. Do these need to be section headings or can we just use the same indented formatting as the setting names? 

![image](https://cloud.githubusercontent.com/assets/14333569/22350490/314fbdce-e3d1-11e6-84c5-3a97e512696c.png)